djangoevents/schema.py

```diff
@@ -8,13 +8,23 @@
 import stringcase
 
 from avro.io import Validate as avro_validate
+from eventsourcing.domain.model.events import DomainEvent
 from .settings import get_avro_dir
+from .utils import classproperty
 from .utils import list_concrete_aggregates
 from .utils import list_aggregate_events
 from .utils import event_to_json
 from .exceptions import EventSchemaError
 
 
+# extend `DomainEvent` with `version` getter
+def get_version(cls):
+    return getattr(cls, '_version', None) or 1
+
+
+DomainEvent.version = classproperty(get_version)
```

Comment:
Not sure whether it's strictly enforced but shouldn't our service use `djangovents.DomainEvent` ? If that is the case a simple inheritance would do the trick.